CMakeLists.txt

```diff
@@ -175,11 +188,19 @@ transform_pc_file("libfaad/faad2.pc.in" "${CMAKE_CURRENT_BINARY_DIR}/faad2.pc" "
 add_executable(faad_cli ${FAAD_SOURCES})
 target_link_libraries(faad_cli faad)
 target_compile_definitions(faad_cli PRIVATE ${FAAD_DEFINES})
-set_target_properties(faad_cli PROPERTIES OUTPUT_NAME faad)
+target_compile_options(faad_cli PRIVATE ${FAAD_FLAGS})
+
+if (MSVC)
+  target_include_directories(faad_cli PRIVATE
+    frontend
+  )
+else()
+  set_target_properties(faad_cli PROPERTIES OUTPUT_NAME faad)
+endif()
 
 # Installation
 
-if(NOT FAAD_BUNDLED_MODE)
+if(NOT FAAD_BUNDLED_MODE AND NOT MSVC)
```

Comment:
Built! 🎉

Reply:
There are some new GCC/Clang warnings, though, so if you could iron them out as well... 😉

Reply:
Sure. Also I would like to run some manual tests to see nothing is broken =)

Reply:
Ready for review